extras/manifests/prometheus-rules.yaml

```diff
@@ -12,15 +12,21 @@ spec:
     rules:
     - alert: OCS_CephClusterErrorState
```

Comment:
Is it a good idea to mention 'OCS' or 'CephCluster' here? 
AFAIU, we are assuming that the user doesn't know/care about the underlying details as long as it provides storage?
I think we should keep alert names generic as this gets listed on the prometheus ui and is expected to be fairly simple. Thoughts?

Reply:
I agree to it, as long as we can filter the alerts from storage system using labels/annotations in UI.
Happy to drop this prefix in that case.

@pcuzner comments?

Reply:
I would prefer to keep these. (Explicit is better than implicit - Zen of Python :P ). If the user isn't concerned about the cluster, I guess they will anyways ignore this. It might be useful for power users. Just a thought.

Reply:
We will filter based on annotations/labels so removing the prefix.